bytes/Dockerfile

```diff
@@ -15,6 +15,7 @@ WORKDIR /app/bytes
 ENV PATH=/home/bytes/.local/bin:${PATH}
 
 RUN mkdir /data && chown bytes: /data
+RUN mkdir ./bytes-data && chown bytes: ./bytes-data
```

Comment:
You can actually set the file mode directly using `mkdir`s `--mode` option

Reply:
I don't understand why an extra data directory is created instead of just using /data? I think we should add a bytes-data volume and mount in on /data in the same way as what we do in the docker-compose.release-example.yml. Then we also don't need to do special things to make sure the directory also exists in the mounted volume and we don't pollute the source repository with data. 

@ammar92 The owner of the directory is being set however, not the mode.

Reply:
@dekkers This is partly a debugging artifact for the particular issue we were solving, but it got me thinking: what if users did not use the `docker-compose-release-example.yml` and still have the default value of `./bytes-data`. Then the build assumes you will run your containers with specific environment variables, which seemed a bit strange unless we explicitly set `ENV BYTES_DATA_DIR=/data` by default.

But given that this data residing in the source repository is mainly a legacy setup, I agree it is a lot nicer to add a bytes-data volume. Then we could add a command that copies all data from the previous default `./bytes-data` to this volume in the case people want to migrate the development docker-compose file?

`docker-compose run --rm --no-deps --entrypoint /bin/sh bytes -c "cp -r ./bytes-data/* /data"`

Reply:
I am not sure whether we really need to provide migrations for this. It's a development setup, so there should only be test data in there?

I agree that we should also change the default value to `/data`.

Reply:
True, although it wouldn't hurt to add a line or two about it in the release notes of course